### DISCUSSION TITLE
Issue using wagmi2

### DISCUSSION THREAD

**Discussion by johnpaulcas** [WalletConnect/walletconnect-monorepo#4142] [Developer Support]
Anyone encountering issue using web3modal3 with wagmi v2 "SyntaxError: The requested module '@web3modal/core' does not provide an export named 'ConstantsUtil"?

Thanks

**Comment by glitch-txs** [comment]
hi, could you share more context? libs, versions, frameworks, code snippet, etc

**Reply by johnpaulcas** [reply]
Hi @glitch-txs, it is resolved now, I need to remove some dependencies from v1, thanks!
